Mathlib/Tactic/Polyrith.lean

```diff
@@ -286,8 +286,11 @@ def sageOutput (args : Array String) : IO SageResult := do
   let path := (← getMathlibDir) / "scripts" / "polyrith_sage.py"
   unless ← path.pathExists do
     throw <| IO.userError "could not find python script scripts/polyrith_sage.py"
-  let s ← IO.Process.run { cmd := "python3", args := #[path.toString] ++ args }
-  match Json.parse s >>= fromJson? with
+  let out ← IO.Process.output { cmd := "python3", args := #[path.toString] ++ args }
+  if out.exitCode != 0 then
+    throw <| IO.userError <|
+      s!"scripts/polyrith_sage.py exited with code {out.exitCode}:\n¬" ++ out.stderr
```

Comment:
Why is there a not symbol here?

Reply:
Because Lean makes it very hard to type `\n`!